TypeCobol/Compiler/Nodes/Data.cs

```diff
@@ -89,9 +90,32 @@ public override bool VisitNode(IASTVisitor astVisitor)
 
 
     public class WorkingStorageSection: DataSection, CodeElementHolder<WorkingStorageSectionHeader>, Parent<DataDefinition>
+    {
+        public WorkingStorageSection(WorkingStorageSectionHeader header) : base(header) { }
+
+        public override IEnumerable<ITextLine> Lines
         {
-	    public WorkingStorageSection(WorkingStorageSectionHeader header): base(header) { }
-	    public override string ID { get { return "working-storage"; } }
+            get
+            {
+                List<ITextLine> lines = new List<ITextLine>();
+                lines.Add(new TextLineSnapshot(-1, "WORKING-STORAGE SECTION.", null));
```

Comment:
Can you try to keep the original case of "working-storage section" ?
Text property of CodeElement should give you the original text written in the source file.